Dockerfile

```diff
@@ -23,12 +23,14 @@ RUN R -e "install.packages('packrat'); packrat::restore()"
 # Add shiny app code
 ADD . .
 
-# Shiny runs as 'shiny' user, adjust app directory permissions
-RUN chown -R shiny:shiny .
-
-# APT Cleanup
-RUN apt-get clean && rm -rf /var/lib/apt/lists/
+# Shiny runs as 'shiny' user, adjust permissions
+RUN chown -R shiny:shiny /var/log/shiny-server && \
+chown -R shiny:shiny /usr/local/lib/R/site-library && \
+chown -R shiny:shiny /var/lib/shiny-server && \
```

Comment:
FYI: At start `shiny-server` creates a `bookmarks` directory within `/var/lib/shiny-server`, hence this change of ownership.